### PR TITLE
Introduce "implicitly restricted APIs".

### PR DESCRIPTION
**Pull request by mfalken**
This expands "Activated-Gated APIs" to cover other reasons that an API
can be implicitly restricted, namely, if it requires system focus or
page visibility.

It also clarifies that prerendering browsing contexts never have system
focus or page visibility, as per discussion in #59 and #55.

***
<a href="https://pr-preview.s3.amazonaws.com/mfalken/alternate-loading-modes/pull/62.html" title="Last updated on May 18, 2021, 11:44 PM UTC (cc0ad5c)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/jeremyroman/alternate-loading-modes/62/9983980...mfalken:cc0ad5c.html" title="Last updated on May 18, 2021, 11:44 PM UTC (cc0ad5c)">Diff</a>

### CODE REVIEW

**Comment by mfalken** [comment]
@domenic can you review this? While writing this I noticed the discussion in #59 is opposite what the current spec says, so we can also continue discussion on #59 if you think it should be "visible".

File: index.bs
Comment (domenic) [inline index.bs:620]: Note that for portals this might not be true, and portals are envisioned to be a type of prerendering browsing context. We might want to state that subtlety here as a TODO or Note, even though portals spec work is kind of on hold while we get these foundations worked out...

File: index.bs
Comment (domenic) [inline index.bs:875]:
```suggestion
- {{WakeLock/request()|navigator.wakeLock.request()}} [[WAKE-LOCK]]
```

**Review by domenic — APPROVED** [review]
LGTM. I agree the existing text about "do not count as hidden" is wrong; maybe I was thinking of portals.

File: index.bs
Comment (mfalken) [reply index.bs]: I made this change, also reverted to [SCREEN-WAKE-LOCK] which I didn't realize the original spec had instead of [WAKE-LOCK]. I don't know what source code that comes from.

File: index.bs
Comment (mfalken) [reply index.bs:620]: Done, added it as an Issue which seemed consistent with the other Issues about portals.

**Comment by mfalken** [comment]
Thanks, done! I added a commit though the diff still seems to show the old one. Would you be able to merge if it's ready? I don't have write access.

**Pull request merged**
